services/QuillLMS/app/models/cron.rb

```diff
@@ -89,6 +91,11 @@ def self.run_at_21_hour_mark
     AdminDiagnosticReports::PerformanceBenchmarkWorker.perform_async
   end
 
+  # 6/7PM depending on daylight savings
```

Comment:
Is there a reason this needs to run specifically at 6pm, or could we move this to run once a day `def self.interval_1_day`? From the description you are trying to avoid other jobs, but I'm not sure we need to do that.